<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open ui File</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>297</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
    <addaction name="reloadAction"/>
   </widget>
   <widget class="QMenu" name="menuPV">
    <property name="title">
     <string>PV</string>
    </property>
    <addaction name="unconnectedAction"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuPV"/>
  </widget>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="reloadAction">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="unconnectedAction">
   <property name="text">
    <string>&amp;Unconnected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
